ci/pipeline.yml

```diff
@@ -161,7 +160,6 @@ jobs:
       - concourse-config/operations/set-garbage-collection.yml
       - concourse-config/operations/base-resource-defaults.yml
       - concourse-config/operations/bosh-dns-aliases.yml
-      - concourse-config/operations/use-bionic.yml
```

Comment:
Just curious what this was and that we don't need it anymore?  Was it something in place while we were transitioning from `xenial`?

Reply:
before 7.3.0, the concourse manifest specified xenial stemcells, and this opsfile overrode that. In 7.3.0 they updated the manifest to specify bionic, so our bionic opsfile now breaks the deployment

Reply:
That's what I figured, but wanted to be sure.  Thanks!